MIGRATION-TO-SCK.md

```diff
@@ -0,0 +1,326 @@
+
+[SCK values.yaml snippet]
+```
+logs:
+  kube-audit:
+    from:
+      file:
+        path: /var/log/kube-apiserver-audit.log
+    timestampExtraction:
+      format: "%Y-%m-%dT%H:%M:%SZ"
+    sourcetype: kube:apiserver-audit
+```    
+
+[SCK-Otel values.yaml snippet]
+```
+extraFileLogs:
+ filelog/kube-audit:
+   include: [/var/log/kube-apiserver-audit.log]
+   start_at: beginning
+   include_file_path: true
+   include_file_name: false
+   resource:
+     service.name: /var/log/kube-apiserver-audit.log
+     host.name: 'EXPR(env("K8S_NODE_NAME"))'
+     com.splunk.sourcetype: kube:apiserver-audit
+```     
+
+You must use the same keyword ("kube-audit" in the above example) for continuing to read from the translated checkpoint data.
+
+### Translating custom configs from SCK to SCK-OTEL for metrics
+#### Limiting/Increasing Application Resources
+If you are using the resources option in SCK to limit/increase the CPU and memory usage in SCK, you can do the same using the resources option in "clusterDataCollector" section in SCK-OTEL.
+
+#### Pod Scheduling configs
+If you are using any pod scheduling operations such as nodeSelector, affinity and tolerations in SCK, you can use the nodeSelector, affinity and tolerations options in SCK-OTEL.
+
+#### Pod Security Context
+If you are using the podSecurityContext option to set a pod security policy, you can use the podSecurityContext option in SCK-OTEL.
+
+#### Custom Annotations
+If you are using the customMetadataAnnotations option to set a custom annotation fields in your logs in SCK, you can use the "annotations" and "podAnnotations" option in SCK-OTEL.
+
+## Step 2: Delete SCK Deployment (Optional)
+If you want to delete your SCK deployment:
+
+* Get your current SCK deployment name by running this command:
+
+  * helm ls
+
+* **If you want to delete only logs and metrics**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete only logs**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete your entire SCK deployment (logs, objects and metrics), run the following command**
+
+  * helm delete local-k8s
+
+## Step 3: Installing SCK-OTEL
+
+* Ensure you have the necessary prerequisites for installing this application, as noted [here](https://github.com/splunk/sck-otel#prerequisites).
+* Set up Splunk to have a Splunk index, HEC and a HEC token ready to use, as noted [here](https://github.com/splunk/sck-otel#setup-splunk).
+* You can use SCK-OTEL as a non-root application on your kubernetes nodes - follow the steps [here](https://github.com/splunk/sck-otel#setup-for-non-root-user-group).
```

Comment:
@dmitryax @rockb1017 @tmorrisonsfx Indeed it is best practice to run SOTelK as non-root user right? can we add the equivalent to https://github.com/splunk/sck-otel#setup-for-non-root-user-group in SOTelK repo?